.github/CHANGELOG.md

```diff
@@ -90,6 +90,7 @@
 1. [ECAM] Fixed gross weight being shown without engines running - @BlueberryKing (BlueberryKing)
 1. [FLIGHT MODEL] Fixes co-pilot not being shown in external view -  @donstim (donbikes#4084)
 1. [BLEED] Replace default bleed system - @BlueberryKing (BlueberryKing)
+1. [ATSU] Use https for SimBrief
```

Comment:
**suggestion**: Add your name to the line.

Reply:
That is an excellent point, 🤣